src/main/scala/is/hail/expr/Relational.scala

```diff
@@ -597,6 +602,33 @@ case class KeyTableLiteral(value: KeyTableValue) extends KeyTableIR {
   def execute(hc: HailContext): KeyTableValue = value
 }
 
+case class ReadKT(path: String,
+  ktType: KeyTableType,
+  localValue: KTLocalValue,
+  dropRows: Boolean,
+  nPartitions: Int,
+  override val partitionCounts: Option[Array[Long]]) extends KeyTableIR {
+
+  val typ: KeyTableType = ktType
+
+  def children: IndexedSeq[BaseIR] = Array.empty[BaseIR]
+
+  def copy(newChildren: IndexedSeq[BaseIR]): ReadKT = {
+    assert(newChildren.isEmpty)
+    this
+  }
+
+  def execute(hc: HailContext): KeyTableValue = {
+    KeyTableValue(typ,
+      localValue,
+      RVD(typ.rowType,
+        if (dropRows)
+          hc.sc.emptyRDD[RegionValue]
```

Comment:
OrderedRVD.empty

Reply:
I'd originally defined an RVD.empty (which maybe should instead by ConcreteRVD.empty), but took it out in favor of moving the if statement one level further in. Would OrderedRVD be the right thing to have here? does it matter?

Reply:
Ah, KT has a RVD not OrderedRVD.  My mistake.  Looking at the code this is a wash right now, but somehow RVD.empty feels like it should be more efficient than RVD(emptyRDD) (esp. compared to an if).

I retract my comment, I'm OK with this as-is or you can put back RVD.empty if you like.

Reply:
I put it back in because I also feel kinda meh about it. I don't think you're getting rid of the if statement either way; it's just a matter of when you're doing the check. Is it really more efficient to do a check in one place vs another?

Reply:
Not really.

I was more thinking about OrderedRVD, where OrderedRVD.empty is WAY more efficient (at the scale of a conditional) than creating an OrderedRVD from a (potentially unordered, even if empty) RDD.  But the scale of a conditional is pretty much irrelevant, so we should do whatever's clearest.  Personally I slightly prefer the intention of expressing an empty RVD.